addons/web_editor/static/src/scss/web_editor.variables.scss

```diff
@@ -202,7 +202,6 @@ $o-we-technical-modal-zindex: 2001;
 }
 
 @mixin o-we-preview-content {
```

Comment:
Seems like this mixin is used for the mediaDialog only (o_video_dialog_iframe)...
It could be removed, let's check if @qsm-odoo  and @rdeodoo are planning to use it somewhere else first

Reply:
(answered in MP, you can delete it)